modules/301-prometheus-metrics-adapter/hooks/pma_server_tls.go

```diff
@@ -16,18 +16,23 @@ limitations under the License.
 
 package hooks
 
-import hooks "github.com/deckhouse/deckhouse/go_lib/hooks/internal_tls"
+import (
+	hooks "github.com/deckhouse/deckhouse/go_lib/hooks/tls_certificate"
```

Comment:
NIT: It seems that the alias does not help to clarify the situation. I'd like to have the tls_certificate here.

Reply:
autorefactor